docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -99,20 +99,30 @@ curl -s $BUNDLE_MANIFEST_URL | yq ".spec.versionsBundles[].eksD.ova.bottlerocket
 
 #### Bottlerocket Template Tags
 
-OS Family - `os:bottlerocket`
+There are two categories of tags to be attached to the Bottlerocket templates in vCenter.
 
-EKS Distro Release
+**os:** This category represents the OS corresponding to this template. The value for this tag must be `os:bottlerocket`.
 
-1.27 - `eksdRelease:kubernetes-1-27-eks-4`
+**eksdRelease:** This category represents the EKS Distro release corresponding to this template. The value for this tag can be obtained programmatically as follows.
 
-1.26 - `eksdRelease:kubernetes-1-26-eks-5`
+Using the latest EKS Anywhere version
+```bash
+EKSA_RELEASE_VERSION=$(curl -sL https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml | yq ".spec.latestVersion")
```

Comment:
Can we find somewhere, maybe on the admin machine or pre-reqs page, where we show how to install yq? or at least call it out as needed?

Reply:
Done